docs/scos/dev/back-end-development/data-manipulation/data-ingestion/spryker-middleware.md

```diff
@@ -22,6 +22,7 @@ redirect_from:
   - /v2/docs/en/spryker-middleware
   - /v1/docs/spryker-middleware
   - /v1/docs/en/spryker-middleware
+  - /spryker-middleware.htm
```

Comment:
Where did you get this redirect? Is it from the broken links report?
It just looks strange.
It seems like it should have ".html" instead of ".htm"